### PR TITLE
Issue #41: Add hungarian translation 

### PR DESCRIPTION
**Pull request by katkodominik**

### CODE REVIEW

File: resources/lang/hu/passwords.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    |--------------------------------------------------------------------------
+    |
+    | The following language lines are the default lines which match reasons
+    | that are given by the password broker for a password update attempt
+    | has failed, such as for an invalid token or invalid new password.
+    |
+    */
+
+    'reset' => 'A jelszavad vissza lett állítva!',
+    'sent' => 'Elküldük az e-mail címedre a jelszó visszaállítási linket!',
```
Comment (luksan47) [inline resources/lang/hu/passwords.php]: nit: Elküldtük

**Comment by katkodominik** [comment]
Should I add print.php in english as well?

**Comment by luksan47** [comment]
You can omit the "print." prefix from the attributes, this will be indicated by filename. So, for example an attribute named "add" in print.php can be accessed with @lang('print.add') or __('print.add').

Also, just noticed one more thing, the translation for validation.php is missing. Could you please also add it in the same style? The `:attribute` tag should not be translated there, this will input the attribute parameter at validation.

Feel free to add the english translations for print too, it's another, smaller issue, assigned it to you.

And thanks for the help!

**Comment by vendi95** [comment]
> You can omit the "print." prefix from the attributes, this will be indicated by filename. So, for example an attribute named "add" in print.php can be accessed with @lang('print.add') or __('print.add').

I think we should stick with the @lang directive in blade templates whenever it is possible.

**Comment by luksan47** [comment]
I just opened an issue to replace @lang() with __(). According to the docs, it might be better.

**Comment by katkodominik** [comment]
> Also, just noticed one more thing, the translation for validation.php is missing.

Where can I find the values for this?

**Comment by luksan47** [comment]
> Where can I find the values for this?
They are at resources/lang/en/validation.php

**Pull request merged**